### ISSUE TITLE
Cannot run chatbot with saved data (Current User)

### ISSUE THREAD

**Issue by iokinpardo** [sintacs/sintacs-mwai-frontend-chatbot-settings#5]
Hi,

I have tried several ways saving the Form data into my Current User data, However I **cannot see - run the chatbot with those params**.
They seem to be saved correctly but not applied to the chatbot. I am required to Save into Original to see those changes.

I like very much the idea of running in a personal way "**Current user**" data as it allows to practice and test them in a 100% safe way, and movig in to **Original (production)** once the user considers they are good enough.

Am I missing something or doing something wrong?

BTW, are those Current User data being saved as **WP User Metadata** or as a **Custom Post Type**? I say this becasue if those are being saved as Custom Post Type we could track saved versions, right? Maybe this is a posible improvement to set as once the User decides to move into Original, there is no way to see previous saved data.

**Comment by sintacs** [comment]
Do you see the changed parameters in the **Queries tab on the AI Engine Settings** page like model, maxTokens, temperature for example? 
![grafik](https://github.com/sintacs/sintacs-mwai-frontend-chatbot-settings/assets/13884907/2c2b3f19-d7d4-4b0b-bf48-a949268f74d8)
If you do not see the Queries tab you can activate it in the Settings tab:
![grafik](https://github.com/sintacs/sintacs-mwai-frontend-chatbot-settings/assets/13884907/329690db-3c17-4026-b2a6-35c14976e807)

What several ways did you try? Saving the form is the only way.

> I like very much the idea of running in a personal way "Current user" data as it allows to practice and test them in a 100% safe way, and movig in to Original (production) once the user considers they are good enough.

Thats exactly how it (should) work(s). To test new settings on user basis.
To be sure, download the plugin again. Before, check the option to delete all settings and uninstall it.

The settings are saved as **WP User Metadata**. 
I like you idea, please post it here: https://github.com/sintacs/sintacs-mwai-frontend-chatbot-settings/discussions/categories/ideas

**Comment by iokinpardo** [comment]
You are completly right. I was just checking that the Chatbot **"AI Names"** and **"User Names"** were not changing on the chatbot. But now as you have said, modifying the **Model** and **Temperature** for example, I have checked that they have been **correctly used on Query Tabs**. So it is running fine.

Maybe it can be **a bit confusing** for users if they change **Visual elements** like "AI Names, User Names, Text Inputs placeholder, ...." and it is not executed on the chatbot. As they cannot check under the AIEngine admin side, they can have a sense that nothing has been changed.

Apart of that it seems to be OK. I will update the idea for setting it as a Custom Post Type instead of User meta.

**Comment by sintacs** [comment]
You are right, this could be a bit confusing not seeing the changes to visual elements take effect. 
Now, with the latest changes, if the chatbot_id is set in the shortcode e.g. [ai_engine_extension_form chatbot_id="default"] the shortcode [mwai_chatbot] gets executed with the attributes aiName, userName, themeId, textInputPlaceholder, textSend, textClear, textInputMaxLength, textCompliance to reflect the user settings in the chatbot.

Thanks for posting your idea.
